### DISCUSSION TITLE
Query Builder Files not Working After 3.09 Update

### DISCUSSION THREAD

**Discussion by Heidinger35** [DaxStudio/DaxStudio#1168] [Q&A]
Hello,

My organization uses query builder files between teams. Version 3.08 fixed a glitch ([1119](https://github.com/daxstudio/daxstudio/issues/1119)) that was not allowing read only users to use Query Builder. Thank you @dgosbell!!

With version 3.09, users are not able to open Query Builder files that were created prior to the update. New query builder files can be shared and reused as expected. However, the teams are concerned that they will need to rebuild all of their query builder files with every update. The error users are receiving is shown below.

![9abaf5de-e747-40ea-bc2b-37f071fc0a79](https://github.com/DaxStudio/DaxStudio/assets/56665555/62884dbd-ced1-4f38-ba20-a0afeb80b6d7)

Is this expected behavior? Can anything be done to resolve this issue? 

Thank you!

**Comment by dgosbell** [comment]
I have not deliberately updated anything and ideally it should work. I definitely do not want to make people re-build files with each release, this is a bug of some sort. I have a preview build available here which I think might fix this issue https://ci.appveyor.com/project/DarrenGosbell/daxstudio/builds/47877700/artifacts . Would it be possible to test this on one of your files and see if it fixes this issue for you?
